packages/colorpickers/src/elements/ColorDialog/index.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useState,
+  useRef,
+  Children,
+  cloneElement,
+  forwardRef,
+  ReactElement,
+  HTMLAttributes
+} from 'react';
+import PropTypes from 'prop-types';
+import mergeRefs from 'react-merge-refs';
+import { Button } from '@zendeskgarden/react-buttons';
+import { GARDEN_PLACEMENT } from '@zendeskgarden/react-modals';
+import Chevron from '@zendeskgarden/svg-icons/src/16/chevron-down-stroke.svg';
+import { ColorPicker, IColorPickerProps } from '../ColorPicker';
+import { IColor } from '../../utils/types';
+import {
+  StyledIcon,
+  StyledButton,
+  StyledFauxInput,
+  StyledDialogPreview,
+  StyledTooltipModal,
+  StyledTooltipBody
+} from '../../styled';
+
+export interface IColorDialogProps extends IColorPickerProps {
```

Comment:
This component is missing the following menu-style props: `eventsEnabled`, `hasArrow`, `isAnimated` (true by default), `popperModifiers`, and `zIndex`.